api/v1alpha1/hostedcluster_types.go

```diff
@@ -650,6 +650,40 @@ type PlatformSpec struct {
 	// +optional
 	// +immutable
 	PowerVS *PowerVSPlatformSpec `json:"powervs,omitempty"`
+
+	// KubeVirt defines KubeVirt specific settings for cluster components.
+	//
+	// +optional
+	// +immutable
+	Kubevirt *KubevirtPlatformSpec `json:"kubevirt,omitempty"`
+}
+
+// KubevirtPlatformSpec specifies configuration for kubevirt guest cluster installations
+type KubevirtPlatformSpec struct {
+	// BaseDomainPassthrough toggles whether or not an automatically
+	// generated base domain for the guest cluster should be used that
+	// is a subdomain of the management cluster's *apps DNS.
+	//
+	// For the KubeVirt platform, the basedomain can be autogenerated using
+	// the *apps domain of the management/infra hosting cluster
+	// This makes the guest cluster's base domain a subdomain of the
+	// hypershift infra/mgmt cluster's base domain.
+	//
+	// Example:
+	//   Infra/Mgmt cluster's DNS
+	//     Base: example.com
+	//     Cluster: mgmt-cluster.example.com
+	//     Apps:    *apps.mgmt-cluster.example.com
```

Comment:
since the wildcard subdomain is under `apps` exactly, i recommend changing:
`*apps` to `*.apps` in all occurrences to make it more evident the url `*.something-apps.mgmt-cluster.example.com` it not valid.

Reply:
fixed